packages/search/src/content/helpers/convert-hub-response.ts

```diff
@@ -85,17 +84,19 @@ function getNextFunction(
 ): () => Promise<IContentSearchResponse> {
   return (auth?: UserSession) => {
     const authentication: UserSession = auth || defaultAuthentication;
-    const headers = authentication
-      ? new Headers({ authentication: JSON.stringify(authentication) })
-      : undefined;
+    const headers =
+      authentication &&
+      authentication.serialize &&
+      new Headers({ authentication: authentication.serialize() });
```

Comment:
this is the non-linting change for this file

Reply:
So - `Headers` will `JSON.stringify()` the object returned from `.searialize()`? Or does that need to be wrapped in `JSON.stringify()`

Reply:
`serialize()` returns a string (in fact, it returns `JSON.stringify(this)`)

Reply:
Ah. that would do it then